_api-reference/cat/cat-segment-replication.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: default
+title: CAT segment replication
+parent: CAT API
+
+nav_order: 53
+has_children: false
+---
+
+# CAT segment replication
+Introduced 2.7
+{: .label .label-purple }
+
+The CAT segment replication operation returns information about active and last completed [segment replication]({{site.url}}{{site.baseurl}}/opensearch/segment-replication/index) events on each replica shard. This API returns metrics on a shard level. The metrics provide information about how far behind the replicas are lagging with respect to the primary shard. 
+
+Call the CAT Segment Replication API only on indexes with segment replication enabled.
+{: .note}
+
+## Path and HTTP methods
+
+```json
+GET /_cat/segment_replication
+GET /_cat/segment_replication/<index>
+```
+
+## Path parameters
+
+The following table lists the available optional path parameter.
+
+Parameter | Type | Description
+:--- | :--- | :---
+`index` | String | The name of the index, or a comma-separated list or a wildcard expression of index names used to filter results. If this parameters is not provided, the response contains information about all indexes in the cluster.
+
+## Query parameters
+
+The CAT segment replication API operation supports the following optional query parameters.
+
+Parameter | Data type | Description
+:--- |:---| :---
+`active_only` | Boolean | If `true`, the response only includes active segment replications. Defaults to `false`. 
+[`detailed`](#additional-metrics-a-detailed-response-returns) | String | If `true`, the response includes additional metrics of each stage of a segment replication event. Defaults to `false`.
+`shards` | String | A comma-separated list of shards to display.
+`format` | String | A short version of the HTTP accept header. Valid values include `JSON` and `YAML`.  
+`h` | String | A comma-separated list of column names to display. 
+`help` | Boolean | If `true`, the response includes help information. Defaults to `false`.
+`time` | Time value | [Units]({{site.url}}{{site.baseurl}}/opensearch/units) used to display time values. Defaults to `ms` (milliseconds).
+`v` | Boolean | If `true`, the response includes column headings. Defaults to `false`.
+`s` | String | Specifies to sort the results. For example, `s=shardId:desc` sorts by shardId in descending order.
+
+## Examples 
```

Comment:
I like the way you formatted these example headings. The examples are easy to follow.